requirements.txt

```diff
@@ -1,8 +1,8 @@
 gunicorn>=19.9.0,<19.9.99
 
 # We're waiting on Django 3 until it's further into the lifecycle.
-Django>=2.2.9,<2.2.9.99
-whitenoise>=4.1.4,<4.1.499
+Django>=2.2.10,<2.2.99
+whitenoise>=4.1.4,<4.1.99
```

Comment:
Even the latest version of whitenoise has latest https://pypi.org/project/whitenoise/5.0.1/

Reply:
See comment above.